aspnetcore/fundamentals/startup.md

```diff
@@ -1,75 +1,119 @@
 ---
-title: Application Startup in ASP.NET Core
+title: Application startup in ASP.NET Core
 author: ardalis
-description: Discover how the Startup class in ASP.NET Core configures services and the application's request pipeline. 
+description: Discover how the Startup class in ASP.NET Core configures services and the app's request pipeline. 
 keywords: ASP.NET Core,Startup,Configure method,ConfigureServices method
 ms.author: tdykstra
 manager: wpickett
-ms.date: 02/29/2017
+ms.date: 12/05/2017
 ms.topic: article
 ms.technology: aspnet
 ms.prod: asp.net-core
 uid: fundamentals/startup
```

Comment:
Since this is a major rewrite, let's tag this as MVC (the doppelgänger, of course): `ms.custom: mvc`